### PR TITLE
Flights

### PR DESCRIPTION
**Pull request by adamwoodcock98**
## Before you start

1. Have you run `git pull origin main` on your branch to bring in the latest changes?
2. Is there an issue on the repository for this request? If not, please create one.
3. Have you followed the correct workflow in the [team documentation](https://docs.google.com/document/d/13yi709ax4nmfTG9J0F57yEFpYRH7fdtczwvVJCrNIsM/edit#)?
4. Conflict resolution to be dealt with **on GitHub**.

## What do these code changes do?
Provide a fully functioning flights page

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough tests.
- [x] I have created a PR ticket on the [Trello board](https://trello.com/b/P8lEsvZe/acebook).
- [x] I will link the PR to the Trello card once created.
- [x] I will send the PR in the Slack channel and ping @here.

### CODE REVIEW

**Comment by adamwoodcock98** [comment]
[Flights #34](https://trello.com/c/b4urkOOn/54-flights-34)

**Review by ruiined — APPROVED** [review]
Nice job 👼

File: client/src/components/Dashboard/flights/addFlight.js [hunk 1]
```diff
@@ -56,6 +58,21 @@ export default function AddFlight({
             }}
             onChange={handleChange}
           />
+          <TextField
```
Comment (heykathl) [inline client/src/components/Dashboard/flights/addFlight.js:61]: Don't need separate field for this. Add date/time together as one field

File: client/src/components/Dashboard/flights/flights.js [hunk 2]
```diff
@@ -36,31 +47,73 @@ const Flights = () => {
...
+          </div>
+        </div>
+        <div className="flights-footer">
+          <AddFlight
+            open={open}
+            handleOpen={handleOpen}
+            handleClose={handleClose}
+            handleChange={handleChange}
+            flight={flight}
+            onSubmit={onSubmit}
```
Comment (heykathl) [inline client/src/components/Dashboard/flights/flights.js:106]: Can we please change to handleSubmit as other components will be using that, for consistency

**Review by heykathl — APPROVED** [review]
Amazing job Adam!!

**Pull request merged**